components/EntryForm.js

```diff
@@ -0,0 +1,72 @@
+import styled from "styled-components";
+import DatePicker from "react-datepicker";
+import { useState } from "react";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default function EntryForm() {
+  const [startDate, setStartDate] = useState(new Date());
+  const [selectedOption, setSelectedOption] = useState("Car");
+
+  const handleDropdownChange = (event) => {
+    setSelectedOption(event.target.value);
+  };
+
+  function handleFormSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    event.target.reset();
+    console.log(data);
+  }
+
+  return (
+    <>
+      <h1 className="calculator-title">Calculator</h1>
```

Comment:
Now you have two ``<h1>``s on your site!

Reply:
Thanks for your comments! I will update my code.